tests/Oryx.BuildImage.Tests/VersionInformationTest.cs

```diff
@@ -432,6 +432,39 @@ public void Python2Alias_UsesVersion_SetOnBenv(string specifiedVersion, string e
                 result.GetDebugInfo());
         }
 
+        [Trait("platform", "go")]
```

Comment:
we should trait the platform as golang and not go .. consistent naming

Reply:
Updated